composer.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "joomla/coding-standard",
+    "description": "Joomla! Coding Standard",
+    "keywords": ["joomla", "coding standard", "phpcs"],
+    "homepage": "https://github.com/joomla/coding-standards",
+    "license": "GPL-2.0+",
+    "require": {
+        "php": ">=5.3.10",
+        "squizlabs/php_codesniffer": "1.*"
```

Comment:
Right now, our standard is not compatible with PHPCS 2 (there's a PR/branch) working on that.  We'd have to do some testing, but I'm fairly sure our standard is compatible with older 1.x releases than PHPCS 1.5; I don't think we should just limit to the last minor branch if that's the case.